### ISSUE TITLE
Where is the Open Token BTN?

### ISSUE THREAD

**Issue by hzx1987227** [OpenWonderLabs/SwitchBotAPI#1]
Hi, 

I can't found the Generate Open Token BTN from the iOS APP.

**Comment by rrense** [comment]
Neither can I find it in the Android app. Is it yet to be released?

**Comment by Minsheng** [comment]
Hey guys! This is Davidson from the dev team. The Generate Token feature is yet to be released in the next app versions for both platforms. If you wanna start playing with it right away, you can submit a request from the app using Feedback and we can generate the token for you as a temporary solution.

**Comment by hzx1987227** [comment]
Thanks for your help!

**Comment by donavanbecker** [comment]
@Minsheng Request sent!!

**Comment by strunker** [comment]
I also sent a request in for token I used the option "ask a question" hopefully that gets routed correctly.

Additionally, is there any official docs at all yet? I spent this past afternoon reversing the mobile app via fiddler proxy. The only thing Ive noticed that changes for the curtain bot, is the turnOff turnOn command, it is likely the same for the switchbots as well.

Whenever we get the token feature we also need to understand how to construct the actual post requests?

turnOff = {"statusCode":100,"deviceType":"WoCurtain","deviceID":"D0XXXXXXXX","command":{"cmdType":"command","deviceCmd":"turnOff","parameter":"default"},"timeStamp":epoch_time,"deviceName":"Bedroom Curtain"}
